vaxrank/report.py

```diff
@@ -51,6 +51,8 @@
     "mhc_alleles",
     "num_somatic_variants",
     "num_coding_effect_variants",
+    "num_variants_with_rna_support",
```

Comment:
One thought: it would be nice to maintain some information for each variant which includes that number of RNA reads before assembling a coding sequence (and other stats we'd like to report here, but in their raw unaggregated form).

Reply:
In general we should move away from this variant/vaccine-peptide strong pairing and aggregate all sorts of info about each variant (including its vaccine peptides), but this is a bigger refactor that should happen soon.